README.rst

```diff
@@ -77,6 +77,6 @@ when you push to GitHub.
 License
 -------
 
-`MIT`_ © 2010-2021 Read the Docs, Inc. & contributors
+`MIT`_ © 2010-2022 Read the Docs, Inc. & contributors
```

Comment:
I believe this isn't required, and we only need the original date: https://hynek.me/til/copyright-years/

```suggestion
`MIT`_ © 2010 Read the Docs, Inc. & contributors
```